languages/elixir/exercises/concept/numbers/lib/assembly_line.ex

```diff
@@ -0,0 +1,11 @@
+defmodule AssemblyLine do
+  @spec production_rate_per_hour(integer()) :: float()
+  def production_rate_per_hour(speed) do
+    :not_implemented
```

Comment:
I think I like just having a comment here. A comment is less distracting than an atom if you've never encountered an atom-like type.

Reply:
Sure, I just wanted to have an equivalent to how many other tracks have an error thrown in these instances, and since there isn't an explicit `NotImplementedError` to raise, I was searching for the erlang equivalent and this came up.

One advantage to this is that if we change the assertion to use `===` then when someone runs `mix test` it will error with the following:

```text
test/assembly_line_test.exs:4
     Assertion with === failed
     code:  assert rate === 0.0
     left:  :not_implemented
     right: 0.0
     stacktrace:
       test/assembly_line_test.exs:6: (test)
```

which may help them to see that they haven't yet implemented it

Reply:
Fair enough. We could also just return the string `"Not Yet Implemented"` or we could raise with `raise "Not Yet Implemented`"

Reply:
I like the `raise "Not Yet Implemented"` idea, it seems explicit enough that it doesn't seem like magic, and helps to get across the idea that the unimplented error is caused by the the statement